### PR TITLE
Neopad keymaps update, added VIA support

### PR DESCRIPTION
**Pull request by rookiebwoy**
## Description

- simplified default keymap
- moved the old default keymap to the new rookiebwoy keymap
- created via keymap
- changed pid and vid in info.json
- updated readme file to better fit the template
- better comments overall

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/neopad/rev1/keymaps/rookiebwoy/keymap.c
Comment (waffle87) [inline keyboards/neopad/rev1/keymaps/rookiebwoy/keymap.c:1]: This directory will need to be removed, as user keymaps are no longer accepted. ([info](https://docs.qmk.fm/#/ChangeLog/20231126)).

Comment (rookiebwoy) [reply keyboards/neopad/rev1/keymaps/rookiebwoy/keymap.c:1]: Sorry, I've missed the changelog. Keymap removed.

**Review by waffle87 — APPROVED** [review]

File: keyboards/neopad/rev1/keymaps/default/keymap.c [hunk 1]
```diff
@@ -16,79 +16,76 @@
...
     ),
 
 };
 
 
 bool encoder_update_user(uint8_t index, bool clockwise) {
-    if (index == 0) {											/* LEFT ENCODER */
+    /* LEFT ENCODER */
+    if (index == 0) {
         switch (get_highest_layer(layer_state)) {
```
Comment (drashna) [inline keyboards/neopad/rev1/keymaps/default/keymap.c:73]: You should be using the Encoder Map feature.

https://docs.qmk.fm/#/feature_encoders?id=encoder-map

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Keyboard updates need to go through `develop`.

File: keyboards/neopad/rev1/keymaps/default/keymap.c (hunk 1 above)
Comment (rookiebwoy) [reply keyboards/neopad/rev1/keymaps/default/keymap.c]: done

**Comment by rookiebwoy** [comment]
If I have another keyboard update like this one, should I push directly through `qmk:develop`?

**Pull request closed without merging**